go/tls/types.go

```diff
@@ -9,6 +9,10 @@ type DigitallySigned struct {
 	Signature []byte `tls:"minlen:0,maxlen:65535"`
 }
 
+func (d DigitallySigned) String() string {
+	return fmt.Sprintf("Signature: Hash=%v Sign=%v Value=%x", d.Algorithm.Hash, d.Algorithm.Signature, d.Signature)
```

Comment:
Could be slightly more explicit about what each bit is in the message, maybe "Signature: Hash Algorithm=%v Signature Algorithm=%v Value=%x" or something?